deploy/config.yml

```diff
@@ -137,3 +237,27 @@ parameters:
     description: Environment name for Setry
 
   - name: ROADMAP_HOST_INVENTORY_URL
+
+  - name: CPU_REQUEST_INVENTORY_SYNC
+    value: 250m
+
+  - name: CPU_LIMIT_INVENTORY_SYNC
+    value: 500m
+
+  - name: MEMORY_REQUEST_INVENTORY_SYNC
+    value: 256Mi
+
+  - name: MEMORY_LIMIT_INVENTORY_SYNC
+    value: 512Mi
```

Comment:
This seems too low but we can set it and see what happens. I would expect to need to bump it.

Reply:
We can set a more sane default or override it later in the deployment config.